hugo/data/categories.yml

```diff
@@ -1,6 +1,37 @@
+#
+# README :
+#
+# It is accepted two different kinds of projects:
+# - without documentation -> the project card will point to the repository source code
+# - having documentation generated by docsrv -> the project card will point to the docs
```

Comment:
I think it's worth linking to docsrv repo. Very nice commenting!

Reply:
Full link, or would it be enough with `src-d/docs`?

Reply:
adding `src-d/docs` link